src/main/kotlin/org/jetbrains/intellij/platform/gradle/plugins/project/IntelliJPlatformBasePlugin.kt

```diff
@@ -315,8 +315,8 @@ abstract class IntelliJPlatformBasePlugin : Plugin<Project> {
             Publishing.register(project, target = intelliJPlatform)
         }
 
-        IntelliJPlatformDependenciesExtension.register(project, target = project.dependencies)
         IntelliJPlatformRepositoriesExtension.register(project, target = project.repositories)
+        IntelliJPlatformDependenciesExtension.register(project, target = project.dependencies)
```

Comment:
May help with https://github.com/JetBrains/intellij-platform-gradle-plugin/issues/1784

Reply:
That doesn't help.